.github/workflows/gh-pages.yml

```diff
@@ -10,15 +10,15 @@ on:
     # Only if docs-related files are touched
     paths:
       - components/eamxx/mkdocs.yml
-      - components/eamxx/docs/*
+      - components/eamxx/docs/**
```

Comment:
Why the double star?

Reply:
Ah, didn't know about the `**` needed to include `/` in the matched pattern. Good to know.

Reply:
I have no clue why that PR triggered the action. Maybe _at the time the PR was issued_ it included a few commits (from another PR) that touched them, but as soon as that PR got merged, they went away?